auth/src/main/java/com/firebase/ui/auth/util/data/PhoneNumberUtils.java

```diff
@@ -0,0 +1,658 @@
+package com.firebase.ui.auth.util.data;
```

Comment:
Here's the diff for this class: http://www.mergely.com/v0dlTATS/. Fixing it was _sooooo_ painful. I literally sat down with the ADB podcast and spent an hour simplifying it until I went insane and discovered this [unbelievably awesome IntelliJ feature](https://www.jetbrains.com/help/idea/structural-search-and-replace.html)—huge cudos to the JetBrains team. Anyway, how can I help you understand the diff?

Reply:
@samtstern Ok, I killed the `PhoneNumberUtils` changes and will move them to another PR so this diff can stay sane. 👍